src/Config/Config.py

```diff
@@ -7,6 +7,10 @@
 '''
     Class to manage config related commands
 '''
+class UsernameNotFound(Exception):
+    def __init__(self, msg):
+        super().__init__(self, msg)
```

Comment:
In the previous review as well I said shouldn't it have a default msg = "User not found"?
Also, kindly make a separate directory for exceptions related to profile and write class over there. Look into "Runner" directory, you can find an example there.